test/smoke/smoke_test.go

```diff
@@ -32,7 +32,9 @@ var LAUNCHPAD = map[string]interface{}{
 
 // configure the network stack
 var NETWORK = map[string]interface{}{
-	"cidr": "172.31.0.0/16",
+	"cidr":                 "172.31.0.0/16",
+	"public_subnet_count":  1,
+	"private_subnet_count": 0,
```

Comment:
again, I don't know why you've added the deprecated variable properties to the smoke test, but if it worked then it worked.